openldap/base.ldif.tpl

```diff
@@ -35,23 +35,23 @@ ou: Policy
 
 # default, Policy, $DOMAIN.$SUFFIX
 dn: cn=default,ou=Policy,dc=$DOMAIN,dc=$SUFFIX
-cn: default policy
+cn: default
 objectClass: pwdPolicy
 objectClass: person
 objectClass: top
 pwdAllowUserChange: TRUE
 pwdAttribute: userPassword
 pwdCheckQuality: 2
-pwdExpireWarning: 600
-pwdFailureCountInterval: 30
+pwdExpireWarning: 259200
+pwdFailureCountInterval: 3600
 pwdGraceAuthNLimit: 5
 pwdInHistory: 5
 pwdLockout: TRUE
 pwdLockoutDuration: 0
-pwdMaxAge: 0
+pwdMaxAge: 7776000
```

Comment:
Can we make this 15552000, 180 days? That's the maximum allowed. 90 days is only a suggestion from OICT, and since password rotation generally means LESS and not MORE security, I'd prefer if we made this period as long as possible.

Reply:
yes